sass/includes/_buttons.scss

```diff
@@ -13,6 +13,11 @@
         font-size: 0.7rem;
     }
 
+    &__icon {
+      height: 13px;
+      width: 13px;
+    }
+
```

Comment:
The `tna-button` class already has support for icons? https://nationalarchives.github.io/tna-frontend/?path=/story/components-button--icon

Reply:
Completely missed that! Thanks, updated 👍